tests/core/models/navigation.js

```diff
@@ -0,0 +1,25 @@
+import Navigation from '../../../src/core/models/navigation';
+
+it('constructs a navigation model from an answers-core vertical results array', () => {
+  const verticalResultsArray = [
+    {
+      verticalKey: 'faqs'
+    }, {
+      verticalKey: 'locations'
```

Comment:
formatting is a little funny, think the open braces should get a new line or the whole object should be a single line

Reply:
going with single line